### PR TITLE
ticdc: refine changefeed scheduler configuration description

### PR DESCRIPTION
**Pull request by ran-huang**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.4 (TiDB 7.4 versions)
- [x] v7.3 (TiDB 7.3 versions)
- [x] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v7.0 (TiDB 7.0 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/14880
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/ticdc-upstream-downstream-check.md [hunk 1]
```diff
@@ -1,18 +1,25 @@
 ---
-title: Data Check for TiDB Upstream and Downstream Clusters
+title: Upstream and Downstream Clusters Data Validation and Snapshot Read
 summary: Learn how to check data for TiDB upstream and downstream clusters.
 aliases: ['/docs/dev/sync-diff-inspector/upstream-downstream-diff/','/docs/dev/reference/tools/sync-diff-inspector/tidb-diff/']
```
Comment (ran-huang) [inline ticdc/ticdc-upstream-downstream-check.md:4]:
```suggestion
aliases: ['/docs/dev/sync-diff-inspector/upstream-downstream-diff/','/docs/dev/reference/tools/sync-diff-inspector/tidb-diff/', '/tidb/dev/upstream-downstream-diff']
```

File: ticdc/ticdc-upstream-downstream-check.md [hunk 2]
```diff
@@ -1,18 +1,25 @@
...
+## Enable Syncpoint
 
-Then, you only need to configure `snapshot` in sync-diff-inspector to verify the data of the TiDB upstream-downstream clusters. The following TiCDC configuration example enables Syncpoint for a created replication task:
+After enabling the Syncpoint feature, you can use [Consistent snapshot read](#consistent-snapshot-read) and [Data consistency validation](#data-consistency-validation).
+
+To enable the Syncpoint feature, set the value of the TiCDC configuration item `enable-sync-point` to `true` when creating a replication task. After enabling Syncpoint, TiCDC writes the following information to the downstream TiDB cluster:
+
+1. During the replication, TiCDC periodically (configured by `sync-point-interval`) align snapshots between the upstream and downstream and saves the upstream and downstream TSO correspondences in the downstream `tidb_cdc.syncpoint_v1` table.
```
Comment (qiancai) [inline ticdc/ticdc-upstream-downstream-check.md:19]:
```suggestion
1. During the replication, TiCDC periodically (configured by `sync-point-interval`) aligns snapshots between the upstream and downstream and saves the upstream and downstream TSO correspondences in the downstream `tidb_cdc.syncpoint_v1` table.
```

File: ticdc/ticdc-upstream-downstream-check.md [hunk 3]
```diff
@@ -1,18 +1,25 @@
...
+2. During the replication, TiCDC also periodically (configured by `sync-point-interval`) executes `SET GLOBAL tidb_external_ts = @@tidb_current_ts`, which sets a consistent snapshot point that has been replicated in backup clusters.
+
+The following TiCDC configuration example enables Syncpoint when creating replication task:
```
Comment (qiancai) [inline ticdc/ticdc-upstream-downstream-check.md:22]:
```suggestion
The following TiCDC configuration example enables Syncpoint when creating a replication task:
```

File: ticdc/ticdc-upstream-downstream-check.md [hunk 4]
```diff
@@ -25,7 +32,25 @@ sync-point-interval = "5m"
...
 ```
 
-## Step 1: obtain `ts-map`
+## Consistent snapshot read
+
+> **Note:**
+>
+> Before you perform consistent snapshot read, make sure that you have [enabled the Syncpoint feature](#enable-syncpoint).
+
+When you need to query the data from the backup cluster, you can set `SET GLOBAL|SESSION tidb_enable_external_ts_read = ON;` in the application to obtain transactionally consistent data on the backup cluster.
```
Comment (qiancai) [inline ticdc/ticdc-upstream-downstream-check.md:41]:
```suggestion
When you need to query the data from the backup cluster, you can set `SET GLOBAL|SESSION tidb_enable_external_ts_read = ON;` for the application to obtain transactionally consistent data on the backup cluster.
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

**Pull request merged**